build.gradle.kts

```diff
@@ -14,22 +14,30 @@ configurations {
   testImplementation { exclude(group = "org.junit.vintage") }
 }
 
-dependencies {
-
-  annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
+repositories {
+  maven { url = uri("https://repo.spring.io/milestone") }
+  mavenCentral()
+}
 
+dependencies {
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("org.springframework.boot:spring-boot-starter-security")
   implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
   implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
-  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:2.0.0-beta-10")
```

Comment:
have you got any queue stuff in this project?  I'd remove if not

Reply:
although I can see lower down that you're using sqs, but weren't using this library previously 😕

Reply:
We use SNS, see SnsService and that uses import uk.gov.justice.hmpps.sqs.HmppsQueueService